### PR TITLE
Add Sans to Tailwind Theme

### PR DESCRIPTION
**Pull request by tregsthedev**
This should add Sans to Tailwind Theme Config, hopefully

### CODE REVIEW

**Comment by KaiDevrim** [comment]
hackclub x undertale!!!!!?!??!?!!?!

File: web/config/tailwind.config.js [hunk 1]
```diff
@@ -27,6 +27,10 @@ module.exports = {
       red: '#ec3750',
       progress: '#ec3750',
     },
+    fontFamily: : {
+      'sans': 'Helvetica, Arial, sans-serif'
+  }
+
```
Comment (sampoder) [inline web/config/tailwind.config.js:33]:
```suggestion
```

File: web/config/tailwind.config.js [hunk 2]
```diff
@@ -27,6 +27,10 @@ module.exports = {
       red: '#ec3750',
       progress: '#ec3750',
     },
+    fontFamily: : {
+      'sans': 'Helvetica, Arial, sans-serif'
```
Comment (sampoder) [inline web/config/tailwind.config.js:31]:
```suggestion
      'sans': `'Baloo 2', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif'`
```

File: web/config/tailwind.config.js [hunk 3]
```diff
@@ -27,6 +27,10 @@ module.exports = {
       red: '#ec3750',
       progress: '#ec3750',
     },
+    fontFamily: : {
```
Comment (sampoder) [inline web/config/tailwind.config.js:30]: It'd be great to also add `mono` and set it to `'SF Mono', 'Roboto Mono', Menlo, Consolas, monospace` as well as adding `header` and set it to `'Shrikhand', system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif`

**Review by sampoder — CHANGES_REQUESTED** [review]
Thanks @tregsthedev! Just a couple of suggestions from me :D

File: web/config/tailwind.config.js (hunk 3 above)
Comment (tregsthedev) [reply web/config/tailwind.config.js:30]: Will do!

**Review by sampoder — APPROVED** [review]
LGTM, thanks @tregsthedev! Can't wait for your next PR.

**Pull request merged**